README.md

```diff
@@ -94,58 +94,58 @@ Verify the deployment by navigating to your server address in your preferred bro
 Or you can use the curl command from your console if your on linux or OSX.  If you choose to use Postman just use the address less the curl command. 
 
 ```sh
-$ curl http://localhost:59993/api/v2/ping
+$ curl http://localhost:59993/api/v3/ping
```

Comment:
We don't need to change the api versions to v3 yet. - I think this file should remain v2.

Reply:
changed those back